.travis.yml

```diff
@@ -12,7 +12,7 @@ addons:
     packages:
       - python2.4
 install:
-  - pip install tox PyYAML Jinja2 sphinx
+  - pip install tox PyYAML Jinja2 sphinx netaddr
```

Comment:
many plugins have extra dependencies, we do not add those to the main ansible ones